src/test/fuzz/versionbits.cpp

```diff
@@ -25,13 +25,13 @@ class TestConditionChecker : public AbstractThresholdConditionChecker
     const Consensus::Params dummy_params{};
 
 public:
-    const int64_t m_begin;
-    const int64_t m_end;
-    const int m_period;
-    const int m_threshold;
-    const int m_bit;
+    const int m_begin = 0;
+    const int m_end = 0;
+    const int m_period = 0;
+    const int m_threshold = 0;
+    const int m_bit = 0;
```

Comment:
Why is this changed?

Reply:
Bad rebase conflict resolution, reverted.